msg/vehicle_command.msg

```diff
@@ -1,109 +1,109 @@
+uint16 VEHICLE_MOUNT_MODE_RC_TARGETING = 3			# Load neutral position and start RC Roll,Pitch,Yaw control with stabilization |
+uint16 VEHICLE_MOUNT_MODE_GPS_POINT = 4				# Load neutral position and start to point to Lat,Lon,Alt |
+uint16 VEHICLE_MOUNT_MODE_ENUM_END = 5				#
 
-uint32 VEHICLE_ROI_NONE = 0                         # No region of interest |
-uint32 VEHICLE_ROI_WPNEXT = 1                       # Point toward next MISSION |
-uint32 VEHICLE_ROI_WPINDEX = 2                      # Point toward given MISSION |
-uint32 VEHICLE_ROI_LOCATION = 3                     # Point toward fixed location |
-uint32 VEHICLE_ROI_TARGET = 4                       # Point toward target
-uint32 VEHICLE_ROI_ENUM_END = 5
+uint16 VEHICLE_ROI_NONE = 0                         # No region of interest |
+uint16 VEHICLE_ROI_WPNEXT = 1                       # Point toward next MISSION |
+uint16 VEHICLE_ROI_WPINDEX = 2                      # Point toward given MISSION |
+uint16 VEHICLE_ROI_LOCATION = 3                     # Point toward fixed location |
+uint16 VEHICLE_ROI_TARGET = 4                       # Point toward target
+uint16 VEHICLE_ROI_ENUM_END = 5
 
-uint32 ORB_QUEUE_LENGTH = 3
+uint8 ORB_QUEUE_LENGTH = 3
 
-float32 param1			# Parameter 1, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float32 param2			# Parameter 2, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float32 param3			# Parameter 3, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float32 param4			# Parameter 4, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float64 param5			# Parameter 5, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float64 param6			# Parameter 6, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float32 param7			# Parameter 7, as defined by MAVLink uint32 VEHICLE_CMD enum.
-uint16 command			# Command ID, as defined MAVLink by uint32 VEHICLE_CMD enum.
+float32 param1			# Parameter 1, as defined by MAVLink uint16 VEHICLE_CMD enum.
+float32 param2			# Parameter 2, as defined by MAVLink uint16 VEHICLE_CMD enum.
+float32 param3			# Parameter 3, as defined by MAVLink uint16 VEHICLE_CMD enum.
+float32 param4			# Parameter 4, as defined by MAVLink uint16 VEHICLE_CMD enum.
+float64 param5			# Parameter 5, as defined by MAVLink uint16 VEHICLE_CMD enum.
+float64 param6			# Parameter 6, as defined by MAVLink uint16 VEHICLE_CMD enum.
+float32 param7			# Parameter 7, as defined by MAVLink uint16 VEHICLE_CMD enum.
+uint16 command			# Command ID, as defined MAVLink by uint16 VEHICLE_CMD enum.
 uint8 target_system		# System which should execute the command
 uint8 target_component		# Component which should execute the command, 0 for all components
 uint8 source_system		# System sending the command
 uint8 source_component		# Component sending the command
 uint8 confirmation		# 0: First transmission of this command. 1-255: Confirmation transmissions (e.g. for kill command)
-uint8 from_external
+bool from_external
```

Comment:
If you change that here, it should be changed in `vehicle_command_ack` as well. And the code should set it to true/false, for example here: https://github.com/dagar/Firmware/blob/2c8da5ae787a520676cf7ea0d297b30096398a75/src/modules/mavlink/mavlink_receiver.cpp#L501

Reply:
Done